### DISCUSSION TITLE
are there any plans to use Mojo to re-implement cocotb?

### DISCUSSION THREAD

**Discussion by muzafferkal** [cocotb/cocotb#3645] [Proposals]
Mojo is a very interesting new language similar to python: https://www.modular.com/max/mojo

It should give significant boost to verification speed as python is a bottleneck right now.

**Comment by marlonjames** [comment]
> It should give significant boost to verification speed as python is a bottleneck right now.

Have you profiled your testbenches and determined specific portions of the Python code base that would benefit from Mojo?
